inc/saiport.h

```diff
@@ -1451,6 +1451,53 @@ typedef enum _sai_port_attr_t
      */
     SAI_PORT_ATTR_DECREMENT_TTL,
 
+    /**
+     * @brief Enable EXP -> TC MAP on port
+     *
+     * Map id = #SAI_NULL_OBJECT_ID to disable map on port.
+     * To enable/disable trust EXP, map ID should be added/removed on port.
+     * Default no map.
+     * If present overrides SAI_SWITCH_ATTR_QOS_EXP_TO_TC_MAP.
+     *
+     * @type sai_object_id_t
+     * @flags CREATE_AND_SET
+     * @objects SAI_OBJECT_TYPE_QOS_MAP
+     * @allownull true
+     * @default SAI_NULL_OBJECT_ID
+     */
+    SAI_PORT_ATTR_QOS_EXP_TO_TC_MAP,
```

Comment:
should we always use MPLS_EXP in the here to make it more clear?

Reply:
> should we always use MPLS_EXP in the here to make it more clear?

sure, i used mpls_exp in defining struct because "exp" is reserved key word :) but for consistency these can all be _MPLS_EXP_

Reply:
thanks. LGTM once you rename it.

Reply:
@lguohan please check if I have addressed your comments